lib/clusters/binaryOutput.js

```diff
@@ -1,8 +1,38 @@
 'use strict';
 
 const Cluster = require('../Cluster');
+const { ZCLDataTypes } = require('../zclTypes');
 
 const ATTRIBUTES = {
+  activeText: { id: 4, type: ZCLDataTypes.string },
+  description: { id: 28, type: ZCLDataTypes.string },
+  inactiveText: { id: 46, type: ZCLDataTypes.string },
+  minimumOffTime: { id: 66, type: ZCLDataTypes.uint32 },
+  minimumOnTime: { id: 67, type: ZCLDataTypes.uint32 },
+  outOfService: { id: 81, type: ZCLDataTypes.bool },
+  polarity: { id: 84, type: ZCLDataTypes.enum8({
+    normal: 0,
+    reverse: 1,
+    })
+  },
+  presentValue: { id: 85, type: ZCLDataTypes.bool },
+  reliability: { id: 103, type: ZCLDataTypes.enum8({
+    noFaultDetected: 0,     // NO-FAULT-DETECTED (0)
+    // noSensor: 1,            // NO-SENSOR (1) - for input clusters only
+    overRange: 2,           // OVER-RANGE (2)
+    underRange: 3,          // UNDER-RANGE (3)
+    openLoop: 4,            // OPEN-LOOP (4)
+    shortedLoop: 5,         // SHORTED-LOOP (5)
+    // noOutput: 6,            // NO-OUTPUT (6) - for input clusters only
+    unreliableOther: 7,     // UNRELIABLE-OTHER (7)
+    processError: 8,        // PROCESS-ERROR (8)
+    // multiStateFault: 9,   // MULTI-STATE-FAULT (9) - for multistate clusters only
+    configurationError: 10  // CONFIGURATION-ERROR (10)
+    })
+  },
+  relinquishDefault: { id: 104, type: ZCLDataTypes.bool },
```

Comment:
Also here, should this be bool or single? (also check other clusters where this is used)

Reply:
No, for Binary it is `bool`: see 3.14.6.4.2